spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -20,49 +20,53 @@ def application_choice
-        hesa_itt_data: {
-          disability: '',
-          ethnicity: '',
-          sex: '',
+        id: application_choice.id.to_s,
+        type: 'application',
+        attributes: {
+          personal_statement: application_choice.personal_statement,
+          hesa_itt_data: {
+            disability: '',
+            ethnicity: '',
+            sex: '',
+          },
+          offer: nil,
+          contact_details: {
+            address_line1: '',
+            address_line2: '',
+            address_line3: '',
+            address_line4: '',
+            postcode: '',
+            country: 'NL',
+            phone_number: '',
+            email: '',
+          },
+          course: {
+            start_date: application_choice.course.start_date,
+            provider_ucas_code: application_choice.provider.code,
+            site_ucas_code: application_choice.course_option.site.code,
+            course_ucas_code: application_choice.course.code,
+          },
+          candidate: {
+            first_name: application_choice.application_form.first_name,
+            last_name: application_choice.application_form.last_name,
+            date_of_birth: application_choice.application_form.date_of_birth,
+            nationality: %w[NL],
+            uk_residency_status: '',
+          },
+          qualifications: [],
+          references: [],
+          rejection: nil,
+          status: application_choice.status,
+          submitted_at: Time.now,
+          updated_at: application_choice.updated_at,
+          withdrawal: nil,
+          work_experiences: [],
         },
-        offer: nil,
-        contact_details: {
-          address_line1: '',
-          address_line2: '',
-          address_line3: '',
-          address_line4: '',
-          postcode: '',
-          country: 'NL',
-          phone_number: '',
-          email: '',
-        },
-        course: {
-          start_date: application_choice.course.start_date,
-          provider_ucas_code: application_choice.provider.code,
-          site_ucas_code: application_choice.course_option.site.code,
-          course_ucas_code: application_choice.course.code,
-        },
-        candidate: {
-          first_name: application_choice.application_form.first_name,
-          last_name: application_choice.application_form.last_name,
-          date_of_birth: application_choice.application_form.date_of_birth,
-          nationality: %w[NL],
-          uk_residency_status: '',
-        },
-        qualifications: [],
-        references: [],
-        rejection: nil,
-        status: application_choice.status,
-        submitted_at: Time.now,
-        updated_at: application_choice.updated_at,
-        withdrawal: nil,
-        work_experiences: [],
       }
     end
 
     it 'returns correct course attributes' do
-      expect(json.dig(:attributes)).to eq expected_attributes
+      expect(json).to eq expected_attributes
```

Comment:
Looks good. 👍 
Could also update the test name here as it's not just course attributes. 
Could be:  `returns correct application attributes`.

Reply:
Good catch

Reply:
Fixed